src/index.ts

```diff
@@ -29,66 +32,53 @@ if ('serviceWorker' in navigator && process.env.NODE_ENV !== 'development') {
 
 window.addEventListener('load', () => {
   const fileInput = document.querySelector('#input') as HTMLElement;
-  const imgElement = document.querySelector('img') as HTMLImageElement;
-  const shareBtn = document.querySelector('#share') as HTMLButtonElement;
+  const mainElement = document.querySelector('main') as HTMLElement;
 
   if (!window.indexedDB || !window.URL) { /* TODO: display error message */ }
 
   const dbPromise = LlamaStorage.create();
-  dbPromise.then((db) => {
-    shareBtn.addEventListener('click', async () => {
-      const id = await db.getCurrentID();
-      if (!id) return; // TODO: show error message
-      const file = await fileFromID(await dbPromise, id);
-      if (!file) return; // TODO: show error message
-      navigator.share({files: [file]});
-    });
-    onDBOpenSuccess(db, imgElement, shareBtn);
+  dbPromise.then(async db => {
+    const id = await db.getCurrentID();
+    if (!id) return;
+
+    const blob = await db.getFile(id);
+    if (!blob) return;
+
+    createOrReplaceVisibleItem(mainElement, dbPromise, id, URL.createObjectURL(blob));
   });
 
-  fileInput.addEventListener('fileselected', (e) => onFileInputChange(e as CustomEvent, dbPromise, imgElement, shareBtn));
+  fileInput.addEventListener('fileselected', e =>
```

Comment:
in general, it might be easier to read if reformatting changes are in separate commits from functional changes, when possible

Reply:
Fair enough. Will keep it as-is in this case, but will avoid this in the future.